core/extension/extension_api_dump.cpp

```diff
@@ -1474,6 +1475,7 @@ static bool compare_dict_array(const Dictionary &p_old_api, const Dictionary &p_
 		if (p_compare_hashes) {
 			if (!old_elem.has("hash")) {
 				if (old_elem.has("is_virtual") && bool(old_elem["is_virtual"]) && !new_elem.has("hash")) {
+					// @todo We should validate hashes here!
 					continue; // No hash for virtual methods, go on.
```

Comment:
The `// No hash for virtual methods, go on.` is probably no longer accurate... Or is this what the `@todo` refers to?